Cartfile.resolved

```diff
@@ -1,5 +1,7 @@
 github "Quick/Nimble" "v5.1.1"
 github "Quick/Quick" "v1.1.0"
-github "ReactiveCocoa/ReactiveCocoa" "5.0.1"
-github "ReactiveCocoa/ReactiveSwift" "1.1.0"
+github "ReactiveCocoa/ReactiveCocoa" "5.0.0-alpha.5"
```

Comment:
Are we certain this alpha version is safe? Shouldn't we update the dependencies on wolmo-core to avoid downgrading?